modules/ROOT/pages/security/proc_pinning-certificates.adoc

```diff
@@ -0,0 +1,47 @@
+== Certificate Pinning
```

Comment:
Haven't used that, so cannot verify this page properly. Pinging @wei-lee to make sure that this is ok